src/devices/src/virtio/vsock/unix/muxer.rs

```diff
@@ -326,30 +326,29 @@ impl VsockMuxer {
             local_port_set: HashSet::with_capacity(defs::MAX_CONNECTIONS),
         };
 
-        // Listen on the host initiated socket, for incomming connections.
+        // Listen on the host initiated socket, for incoming connections.
         muxer.add_listener(muxer.host_sock.as_raw_fd(), EpollListener::HostSock)?;
         Ok(muxer)
     }
 
     /// Handle/dispatch an epoll event to its listener.
-    fn handle_event(&mut self, fd: RawFd, evset: EventSet) {
+    fn handle_event(&mut self, fd: RawFd, handler_evset: EventSet) {
```

Comment:
This is very subjective, but I don't like the naming `handler_evset`. Unfortunately, I couldn't think of a better one than `event_set`, which is basically just the longer version.

The problem is that it's not the event of the handler (`handler_evset`), it's the event set that the calling functions want to handle, so it would be something like `handled_evset`, which is weird and generally would be shortened to just `evset`.

I propose you name it `event_set` in order to avoid the shadowing of the other variable and just be done with it.

Reply:
Thanks for the feedback, I'll address this asap! :+1: